pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -121,6 +121,12 @@
     public static final String SEGMENT_RELOCATOR_INITIAL_DELAY_IN_SECONDS =
         "controller.segmentRelocator.initialDelayInSeconds";
 
+    // configs for uploading missing LLC segments copy to segment store
+    public static final String ENABLE_UPLOAD_MISSING_LLC_SEGMENT_TO_SEGMENT_STORE =
```

Comment:
nit: ENABLE_DEEP_STORE_LLC_SEGMENT_CHECK ? I don't know, I am just thinking of some shorter name here.